scripts/pod_lib_lint.rb

```diff
@@ -42,13 +48,28 @@ def main(args)
 
   command = %w(bundle exec pod lib lint --sources=https://cdn.cocoapods.org/)
 
+  # Split arguments that need to be processed by the script itself and passed
+  # to the pod command.
+  script_args = {}
```

Comment:
@wilhuff I invited you to provide the feedback on the `pod_lib_lint.rb` and `.travis.yaml` changes.